mapie/tests/test_conformity_scores.py

```diff
@@ -382,18 +382,17 @@ def test_residual_normalised_prefit_get_estimation_distribution() -> None:
 @pytest.mark.parametrize("score", [AbsoluteConformityScore(),
                                    GammaConformityScore(),
                                    ResidualNormalisedScore()])
-@pytest.mark.parametrize("alpha", [[0.3], [0.5, 0.4]])
+@pytest.mark.parametrize("alpha", [[0.5], [0.5, 0.6]])
 def test_intervals_shape_with_every_score(
     score: ConformityScore,
     alpha: Any
 ) -> None:
+    estim = LinearRegression().fit(X_toy, y_toy)
```

Comment:
Why did we switch `cv="prefit"`?

Reply:
Because we don't have enough data for fit and calibrate methods (because of the change in the quantile formula). Because here we are simply checking the shape of the output, we don't care which method we use.